src/utils/get-global-config.js

```diff
@@ -29,17 +29,18 @@ const getGlobalConfigOnce = async function () {
 }
 
 const getGlobalConfig = async function () {
-  const retries = 3
-  for (let retry = 1; retry <= retries; retry++) {
+  const maxAttempts = 3
+  const retry = async (retriesLeft) => {
```

Comment:
Can we keep the old code? It seems more straightforward that using recursion

Reply:
Alright.